### PR TITLE
Sign entries for cosign also for sign new manifests and tag docker

### PR DESCRIPTION
**Pull request by midnightercz**

### CODE REVIEW

File: pubtools/_quay/tag_docker.py
Comment (emilyzheng) [inline pubtools/_quay/tag_docker.py:1]: Could you also update it in merge_manifest_lists_sign_images?

File: pubtools/_quay/tag_docker.py [hunk 1]
```diff
@@ -524,6 +525,19 @@ def copy_tag_sign_images(self, push_item: Any, tag: str, executor: Executor) ->
             raise ValueError("Tagging workflow is not supported for multiarch images")
 
         if push_item.claims_signing_key:
+            # add entries in internal format for cosign
+            to_sign_entries_internal.append(
+                SignEntry(
+                    repo=repo,
+                    reference="quay.io/"
+                    + get_internal_container_repo_name(list(push_item.repos.keys())[0])
```
Comment (emilyzheng) [inline pubtools/_quay/tag_docker.py:533]: Quay namespace is missing.

File: pubtools/_quay/tag_docker.py [hunk 2]
```diff
@@ -666,7 +697,7 @@ def merge_manifest_lists_sign_images(
                     if outdated_manifests:
                         signer.remove_signatures(outdated_manifests, _exclude=current_signatures)
                     signer.sign_containers(
-                        to_sign_entries,
+                        to_sign_entries if signercls.pre_push else to_sign_entries_internal,
```
Comment (emilyzheng) [inline pubtools/_quay/tag_docker.py:700]: It should always be to_sign_entries as it's already in if pre_push is true. And line #733 needs to be to_sign_entries_internal.

Comment (midnightercz) [reply pubtools/_quay/tag_docker.py]: fixed

**Review by emilyzheng — APPROVED** [review]

**Review by querti — APPROVED** [review]

**Pull request merged**